agent/php_newrelic.h

```diff
@@ -593,6 +593,8 @@ nrinibool_t
 nrinibool_t
     vulnerability_management_package_detection_enabled; /* newrelic.vulnerability_management.package_detection.enabled
                                                          */
+nrinibool_t
+    vulnerability_management_composer_api_enabled; /* newrelic.vulnerability_management.composer_api.enabled */
```

Comment:
I couldn't see any unit/integration tests where `vulnerability_management_package_detection_enabled;` is disabled but `newrelic.vulnerability_management.composer_api.enabled` is enabled.  Would be good to have a couple of tests.

Reply:
Added in f6b438aa1a1ffcb7b693d27cf0aed1f7a56b895d.